apps/frontend/src/components/WorkoutMenu/WorkoutEditor.tsx

```diff
@@ -77,16 +65,21 @@ export const WorkoutEditor = ({
   const { user, saveLocalWorkout } = useUser();
   const toast = useToast();
   const token = (user.loggedIn && user.token) || null;
+  const [showDeleteDialogue, setShowDeleteDialogue] = React.useState(false);
```

Comment:
```suggestion
  const [showDeleteDialog, setShowDeleteDialog] = React.useState(false);
```

Reply:
@sivertschou Oh, mercy me! I forgot that we were in the colonies